composer.json

```diff
@@ -26,7 +26,7 @@
         }
     },
     "require": {
-        "php": "^7.4",
+        "php": "^7.4 || ^8.0",
```

Comment:
Did we get a request to allow for 8.0? Should we bump this to an 8.0 requirement at some point?

Reply:
> Did we get a request to allow for 8.0?

Not explicitly, but since we are now supporting it for clients, I might want to be able to run tests in multiple versions.

> Should we bump this to an 8.0 requirement at some point?

Yeah, possibly. This will take a little coordination as we will need to ensure the bazel build pipeline has the proper PHP version(s).